pylhc_submitter/submitter/htc_utils.py

```diff
@@ -58,25 +58,37 @@
 NOTIFICATIONS = ("always", "complete", "error", "never")
 
 
-COLUMN_SHELL_SCRIPT = "ShellScript"
-COLUMN_JOB_DIRECTORY = "JobDirectory"
-COLUMN_JOB_FILE = "JobFile"
-
 
 # Subprocess Methods ###########################################################
 
 
-def create_subfile_from_job(cwd: Path, job: str):
-    """Write file to submit to ``HTCondor``."""
+def create_subfile_from_job(cwd: Path, submission: Union[str, htcondor.Submit]) -> Path:
+    """
+    Write file to submit to ``HTCondor``.
+    
+    Args:
+        cwd (Path): working directory
+        submission (str, htcondor.Submit): HTCondor submission definition (i.e. content of the file)
+
+    Returns:
+        Path: path to sub-file
+

```

Comment:
Empty line here, don't know if it's on purpose but you don't have it in some other functions.